_sources/Sequences/week2a1.rst

```diff
@@ -220,8 +221,10 @@ Chapter Assessment
     class myTests(TestCaseGui):
 
       def test_output(self):
+	import re
```

Comment:
Same as above, but this looks like the indentation is wrong.  Please make sure that you build the book and test any code changes you make to be sure that you are not introducing syntax errors.